srcpkgs/kpackage/template

```diff
@@ -13,11 +13,6 @@ homepage="https://invent.kde.org/frameworks/kpackage"
 distfiles="${KDE_SITE}/frameworks/${version%.*}/${pkgname}-${version}.tar.xz"
 checksum=6cdb5e6aeb8f45dcd0cef4863edb698cee0fdd216171c00aa5acf5c75cdc1f1d
 
-post_install() {
```

Comment:
These lines cause a cross-compile configure failure in kcm-wacomtablet, but I'm not sure why they were added (it was done in 59860865e75, but the commit message does not provide an explanation for what these lines do).

Reply:
@sgn since you made that commit, could you provide some context please?

Reply:
See the commit message ;) Did you try adding `kpackage` to `hostmakedepends`?